backend/src/index.js

```diff
@@ -140,13 +140,8 @@ function startStatsAggregation() {
 async function main() {
   console.log("Starting Explorer backend...");
 
-  try {
+  if (models.sequelizeTelemetryBackend) {
```

Comment:
Why won't this be throwing errors anymore?

Reply:
If I'm not mistaken that's because now it will switch to public credentials automatically which prevents throwing annoying errors to console

Reply:
We only needed to catch an error here since we used the same credentials for reads and writes to telemetry database and this code attempted to create tables with read-only credentials (unless you configure it with private credentials with write access), and we simply ignored that failure since the assumption was that if we cannot create the telemetry tables, we won't be able to store the telemetry data, but that is not critical since on dev server you won't get telemetry requests anyway.

Now, in the contrast, we can simply skip even trying to initialize telemetry database if the write credentials are not set. Thus, we don't throw any scary errors at developers.